release/pkg/versions.go

```diff
@@ -116,3 +87,17 @@ func newCliVersioner(cliVersion, pathToProject string) *cliVersioner {
 func (v *cliVersioner) patchVersion() (string, error) {
 	return v.cliVersion, nil
 }
+
+func GenerateComponentChecksum(hashes []string) string {
+	b := make([][]byte, len(hashes))
+	if hashes != nil {
+		for i, str := range hashes {
+			b[i] = []byte(str)
+		}
+	}
+	joinByteArrays := bytes.Join(b, []byte(""))
+	sum256 := sha256.Sum256(joinByteArrays)
+	sumStr := string(sum256[:])[:4]
```

Comment:
curious why 4?

Reply:
Using 4 gives us the first 8 characters from the hash. so it'll look something like `v1.0.1+a1d86492`

There's no hard reason on why I chose this number. Open to suggestions